static/js/components/InstrumentPage.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Typography from "@material-ui/core/Typography";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import NewInstrument from "./NewInstrument";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    maxWidth: "22.5rem",
+    backgroundColor: theme.palette.background.paper,
+  },
+  paperContent: {
+    padding: "1rem",
+  },
+}));
+
+export function instrumentTitle(instrument, telescopeList) {
+  const telescope_id = instrument?.telescope_id;
+  const telescope = telescopeList?.filter((t) => t.id === telescope_id)[0];
+
+  if (!(instrument?.name && telescope?.name)) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+
+  let result = `${instrument?.name}/${telescope?.nickname}`;
+
+  if (instrument?.filters || instrument?.api_classname) {
+    result += "(";
+    if (instrument?.filters) {
+      result += `filters: ${instrument.filters}`;
+    }
+    if (instrument?.api_classname) {
+      result += ` / API Classname: ${instrument?.api_classname}`;
+    }
+    result += ")";
+  }
+
```

Comment:
I would put the name of the instrument/telescope in a separate div from the other info, and maybe make that div in bold or something. It would make it much more readable.

Reply:
see if you like the update!